### ISSUE TITLE
Unpack operator isn't properly evaluated

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#279]
Example:
```
def foo():
    rest = (4, 5, 6)
    t = 1, 2, 3, *rest
    return t
```
Type of "t" should be Tuple[1, 2, 3, 4, 5, 6] but is instead reported as Type[1, 2, 3, 4].

**Comment by erictraut** [comment]
This will be fixed in the next version.

**Comment by erictraut** [comment]
This is now fixed in 1.0.70, which I just published.
